#request
int32 x_coord
int32 y_coord
int32 z_coord
---
#response
bool success

